source/addComment.js

```diff
@@ -1,3 +1,39 @@
 $(document).ready(function(){
-  // add your code here
-})
+
+  $(document.body).append(
```

Comment:
so you're appending this to the body right away. what if only append it when the button is clicked? then, we'd need to use event delegation to submit the new comment 🎉 wooooo event delegation!